test/readme.md

```diff
@@ -143,6 +144,18 @@ modify the steps to work with existing Node projects.
 
 > **Note:** When making driver changes, you will need to run `npm run build:ts` with each change in order for it to take effect.
 
+## Writing Tests
+
+> TODO: flesh this section out more
```

Comment:
are you intentionally leaving the TODO here?

Reply:
There's a lot more that could be written here about how to write good tests / how our to go about fitting into the way we write them, I could file follow up but I'd want to be specific if I made tickets. Like reccomending hooks for client creation and cleanup, or table tests when possible, or minimize assertions to keep a test clear. Does that all sounds like something that would fit here? 

My short term goal was to capture these hazards at least somewhere

Reply:
I think it's a good idea to do in the future, but this is fine for now